core/schemas.py

```diff
@@ -18,6 +18,7 @@ def __init__(self, *, only: Optional[StrSequenceOrSet] = None, exclude: StrSeque
         super().__init__(only=only, exclude=exclude, many=many, context=context, load_only=load_only,
                          dump_only=dump_only, partial=partial, unknown=unknown)
 
+    @post_load()
```

Comment:
If we don't add `@post_load` on the contract, the decorator is able to add it but it'll never get triggered, I'm not entirely sure why this happens